ral-facilities/datagateway
# Preventing 'null' from displaying in descriptions and summaries

## Description
This fixes an issue where 'null' would display on the page in cases where an investigation, dataset or datafile description was being displayed on the page but this variable wasn't actually set. Usually this means it was never provided with the data to begin with so we now show a more helpful message: 'Description not provided'.

## Testing instructions
Browse various investigations and datasets to find instances of 'Description not provided'. Expand the 'More Information' segment as well to ensure this message is displayed there too. It may help to manually edit the passed-in data so that no description value is provided but the snapshots test this themselves.


## Agile board tracking
Closes #715
